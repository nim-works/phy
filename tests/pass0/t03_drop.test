discard """
  output: "(Done)"
"""
(TypeDefs
  (ProcTy (Void))
)
(GlobalDefs)
(ProcDefs
  (ProcDef (Type 0) 0 (Locals)
    (List
      (Block (Params)
        (Drop (IntVal 8))
        (Return)
      )
    )
  )
)