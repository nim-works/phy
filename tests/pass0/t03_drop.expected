.type t0 () -> void
.start t0 p0
  LdImmInt 8
  Drop
  Ret
.end
